.github/workflows/helm.yml

```diff
@@ -39,38 +35,58 @@ jobs:
     outputs:
       charts_path: ${{ steps.set-path.outputs.path }}
       repository_name: ${{ steps.set-path.outputs.repo }}
-  lint:
-    name: Lints
-    runs-on: ubuntu-latest
-    needs: prepare-charts-path
-    steps:
-        - uses: actions/checkout@v4
-        - uses: azure/setup-helm@v3.5
-          with:
-            version: "v3.12.0"
-          id: install
-        - name: Build dependencies
-          run: helm dep update ${{ needs.prepare-charts-path.outputs.charts_path }}
-        - name: Lint Chart
-          run: helm lint ${{ needs.prepare-charts-path.outputs.charts_path }}
```

Comment:
Why do we remove the linting?

Reply:
Moved and merged w/ into `tests` job:
https://github.com/Substra/substra-gha-workflows/blob/fix/chart-publication/.github/workflows/helm.yml#L70-L71

Which is similar to how it was previously done (cf: [build](https://github.com/Substra/substra-backend/actions/runs/7411551152/job/20166335097))

Reply:
Ok, but it looks that we are loosing parallelism between the tasks we could not immediately tell if it's a linting issue, changelog issue or a testing issue) I don't know if it's an issue, tho.

Reply:
Indeed we are loosing parallelism but I don't think that is an issue because:
1. All setup steps (checkout, install, build, ...) are now performed only once for all checks
2. Workflow feedback is still pretty fast (23s failure [here](https://github.com/Substra/substra-frontend/actions/runs/8067016816/job/22036462552)) - even if it might be not exhaustive enough

---

I can look into trying to perform all the checks even if the previous one failed. Would that be ok?

Reply:
I think the solution you proposed is fine, just keeping it in mind and adapt later if needed :)